themes/vuetober/assets/css/about.abeee0df.css

```diff
@@ -1 +1 @@
\ No newline at end of file
+.cont-color[data-v-a35b3b7c]{height:80vh;width:97.5vw;background:linear-gradient(45deg,#5352f6,#8381fd);border-radius:0 0 0 10rem;position:absolute;z-index:-10}.content-container[data-v-a35b3b7c]{padding-top:15rem;max-width:500px;min-width:430px;position:absolute;z-index:100}.cont-img-pic[data-v-a35b3b7c]{height:80vh;-webkit-transform:translateX(0) translateY(20%);transform:translateX(0) translateY(20%)}input.form-control[type=password][data-v-6825c358]:focus:not([readonly]),input.form-control[type=text][data-v-6825c358]:focus:not([readonly]){border-bottom:1px solid #5352f6;-webkit-box-shadow:0 1px 0 0 #5352f6;box-shadow:0 1px 0 0 #5352f6;border-radius:0}input[data-v-6825c358]{border:none;border-radius:0;border-bottom:1px solid grey;-webkit-box-shadow:none;box-shadow:none}.test2[data-v-19fd45c3]{color:#5352f6}.cont_img[data-v-19fd45c3]{background:linear-gradient(45deg,#5352f6,#8381fd);border-radius:150px 0 150px 150px;z-index:-1}.cont_img_pic[data-v-19fd45c3]{height:80vh;-webkit-transform:translateX(-40%) translateY(20%);transform:translateX(-40%) translateY(20%)}.cont_img[data-v-16b21aff]{background:linear-gradient(45deg,#5352f6,#8381fd);border-radius:150px 0 150px 150px;z-index:-1}.cont_img_pic[data-v-16b21aff]{height:75vh;-webkit-transform:translateX(-25%) translateY(15%);transform:translateX(-25%) translateY(15%)}.test2[data-v-16b21aff]{color:#5352f6}.teacher-name[data-v-aa0b8de2]{font-size:.875rem}input.form-control[type=text][data-v-aa0b8de2]:focus:not([readonly]){border-bottom:1px solid #5352f6;-webkit-box-shadow:0 1px 0 0 #5352f6;box-shadow:0 1px 0 0 #5352f6;border-radius:1px}.bg-search[data-v-aa0b8de2]{background-color:#f3f5f8}.test2[data-v-aa0b8de2]{border:none;border-bottom:1px solid #d7d7e8}.list-group-item.active[data-v-aa0b8de2],.list-group-item.active[data-v-aa0b8de2]:focus,.list-group-item.active[data-v-aa0b8de2]:hover{z-index:2;color:#5352f6;background-color:#d7d7e8;border-color:#d7d7e8}.teachers[data-v-aa0b8de2]{height:81vh;overflow:auto}.feeder[data-v-696eaf7d]{height:40vh;overflow:auto}.searchbg[data-v-ba7e7c96]{background-color:#fff;height:100vh}.teacherbg[data-v-ba7e7c96]{background-color:#f8f5f3;height:100vh}.bg-all[data-v-5fc21618]{background-color:#5352f6}.form1[data-v-5fc21618]{width:30%;background-color:#f4f5f8;border-radius:0;border:none}.textfield[data-v-5fc21618]{border-radius:0}
```

Comment:
Dajte si `vutober/assets` cele do `.gitignore`, automaticky generovane subory by nemali byt v gite

Reply:
Teda neviem ci uplne vsetko v assets je generovane, tie pngcka mozno by mali ostat v gite, take mozete dat do .gitignore len `css` a `js`

Reply:
Len potom sa to dava zlozitejsie na server. Docasne takto clonnem cely repozitar a hned to funguje.